### ISSUE TITLE
Perfect scrollbar working even bootstrap modal is opened

### ISSUE THREAD

**Issue by garramja** [creativetimofficial/material-dashboard-pro-angular#131]
Hi,

When bootstrap modal with large content is opened, perfect scrollbar is still active at background. If you scroll with mouse background is scrolling but not modal.

How can this be fixed?

First movement is made with mousse wheel. 
Second movement is made with scrollbar of modal overlay.

![modal bug](https://user-images.githubusercontent.com/19532063/32365628-d22c4d8a-c07a-11e7-8b2d-28e934ca60eb.gif)

**Comment by drenda** [comment]
Annoying problem. I think that is present also with tabs. I hope there is a fix for that

**Comment by chelaruc** [comment]
@garramja @drenda Sorry for the late response! Have you found a solution for this? If not, I'll try to fix it in v1.5.0.

**Comment by drenda** [comment]
No solution for me.

**Comment by chelaruc** [comment]
@garramja @drenda Sorry for the late response. TO fix that you need to add this on `notifications.component.ts`:
```
ngOnInit(){
        var mainPanel = document.getElementsByClassName('main-panel')[0];
        $('.modal').on('shown.bs.modal', function () {
          mainPanel.classList.add('overflow');
        })
        $('.modal').on('hidden.bs.modal', function () {
          mainPanel.classList.remove('overflow');
        })
    }
```
and in `src/assets/sass/material-dashboard/_dialogs.scss`:
```
.overflow{
  overflow: hidden;
}
```
This will be fixed in v1.5.0.

Best, 
Ciprian

**Comment by Ananya-96** [comment]
@chelaruc  i tried your solution but it did not work. Instead tried
` 
var mainPanel = document.getElementsByClassName('main-panel')[0];
     $('.modal').on('shown.bs.modal', function () {
        mainPanel.style.height = "auto";
      mainPanel.style.maxHeight ="none";
    })
    $('.modal').on('hidden.bs.modal', function () {
        mainPanel.style.height="100%"
      mainPanel.style.maxHeight = "100%"
    })`

which seems to solve my problem.
